strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/login/LoginControllerTest.java

```diff
@@ -34,7 +34,7 @@ public void shouldReturnGeneratedToken()
     {
         LoginInput loginInput = new LoginInput();
         loginInput.setUsername("admin");
-        loginInput.setPassword("password");
+        loginInput.setPassword("$2a$10$WqtVx7Iio0cndyR1lEaKW.SWhUYmF/zHHG5hkAXvH5hUmklM7QfMO");
```

Comment:
Why is this a hash? The idea here is that you actually submit JSON to the controller with the raw user credentials as this is what you would receive from the login form?